templates/command/command.go.tmpl

```diff
@@ -1,79 +1,74 @@
 {{- $packageName := .PackageName -}}
-package {{$packageName}}
+package {{ $packageName }}
```

Comment:
Was this auto-formatted?  If so, let's see if we can all use the same formatter.

Reply:
No. But it seemed to make sense that if we use `- -` to control whitespace `{{- $someVar -}}` that we should always have spaces between the reference and the curly braces. The whitespace didn't seem to get removed when there was no space between the `-` and the reference.